src/components/NewRenter/messages.ts

```diff
@@ -1,6 +1,21 @@
 import { defineMessages } from 'react-intl';
 
 export default defineMessages({
+  catalog: {
+    id: 'Catalog.pageTitle',
+    defaultMessage: 'Catalog',
+    description: 'Title for the page header',
+  },
+  notFound: {
+    id: 'NotFound.pageTitle',
+    defaultMessage: 'Not Found',
+    description: 'Title for the page header',
+  },
+  newRenter: {
+    id: 'NewRenter.pageTitle',
+    defaultMessage: 'Create a new ad',
+    description: 'Title for the page header',
```

Comment:
Perhaps, description we should remove everywhere

Reply:
I guess we can use selectively, just for those fields which require additional explanation.